requirements.txt

```diff
@@ -17,7 +17,7 @@ future==0.17.1            # via cookiecutter
 idna==2.8                 # via requests
 isort==4.3.10             # via pylint
 jinja2-time==0.2.0        # via cookiecutter
-jinja2==2.10              # via cookiecutter, jinja2-time
+jinja2==2.10.1
```

Comment:
You should NOT modify ``requirements.txt`` but ``requirements.in`` and rerun pip-tools (e.g. using https://github.com/ITISFoundation/dockerfiles/tree/master/pip-kit)

in requirements.in you should add the constraint ``jinja2>=2.10.1``

Reply:
Try using pip-kit and if it is not easy to use or unclear, please let me know. By using the tools we get them fully tuned to our needs :-)

Reply:
hmm which is what I did, see [requirements.in](requirements.in)

Reply:
OK. My bad I did not see the comment  after jinja entry  i.e. ``# via cookiecutter, jinja2-time`` and thought it was modified by hand